### PR TITLE
Adds Locale Updation Script + Adds Locale Data +  Localises 4 new strings + Initial Text in Input Boxes in apps

### PR DESCRIPTION
**Pull request by smkvr98**
Script for Locale updation -  tools/refreshLocaleFiles.py
Data for localisation - SSoT @ public/locales/UI-localisation.csv, derived by locales/translation.{}.json 
Localises Initial Text in Count, Sort & Converter. 
4 new strings - diffView headers + tooltips. 

(updation of xx.json is presently copying en.json and replacing  ```: "``` with ```: "^``` and ```"[(,\n)]``` with ```\$$0``` before ```\$``` with ```$``` ).

### CODE REVIEW

File: src/pages/WordFrequency/components/DiffView/DiffView.tsx [hunk 1]
```diff
@@ -65,42 +67,45 @@ export const DiffView = (props: DiffViewProps) => {
     fetchData()
   }, [nodeRelativePath, script])
 
-  const columnDefinitions: KSCUI.C.KsTableColumnDefinition[] = [
```
Comment (parthopdas) [inline src/pages/WordFrequency/components/DiffView/DiffView.tsx:68]: general suggestion is to try a reduce the diffs in changes. 

it is not just so the PR is easier but also make is easy (and therefore more likely) that the author can ensure only changes he intended go in.

i thin

File: src/pages/WordFrequency/components/DiffView/DiffView.tsx [hunk 2]
```diff
@@ -65,42 +67,45 @@ export const DiffView = (props: DiffViewProps) => {
     fetchData()
   }, [nodeRelativePath, script])
 
-  const columnDefinitions: KSCUI.C.KsTableColumnDefinition[] = [
-    {
```
Comment (parthopdas) [inline src/pages/WordFrequency/components/DiffView/DiffView.tsx:69]: in this case it should be easy to just modify line 68 to add another lambda and keep the rest same.

File: tools/refreshLocaleFiles.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import json
```
Comment (parthopdas) [inline tools/refreshLocaleFiles.py:1]: nice - well done.

**Review by parthopdas — COMMENTED** [review]
for the ci changes, python will be present on the machine.

you can replicate lines 27-29 in a step just before https://github.com/digitalpalitools/web-ui/blob/master/.github/workflows/cd.yml#L27 and then call python instead of yarn.

File: public/locales/translation.en.json [hunk 4]
```diff
@@ -1,43 +1,47 @@
-{
```
[the remark this answers is not in the document]
Comment (smkvr98) [reply public/locales/translation.en.json:1]: The keys were sorted.

File: .github/workflows/cd.yml [hunk 5]
```diff
@@ -24,6 +24,9 @@ jobs:
       - name: Install
         run: yarn --no-progress --non-interactive --no-lockfile
 
+      - name: Update locale files
+        run: python3 tools/refreshLocaleFiles.py
```
Comment (parthopdas) [inline .github/workflows/cd.yml:28]: Did it work? Do the ci logs show it working as expected?

**Review by parthopdas — APPROVED** [review]

File: .github/workflows/cd.yml (hunk 5 above)
Comment (smkvr98) [reply .github/workflows/cd.yml:28]: The script doesn't print anything. Must I be adding something on the lines of a verbose mode?

Comment (smkvr98) [reply .github/workflows/cd.yml:28]: The script now prints out messages as it updates locales.

**Pull request merged**